warehouse/schema/schema.go

```diff
@@ -100,7 +99,7 @@ func New(
 		"destinationId": warehouse.Destination.ID,
 	})
 	if conf.GetBoolVar(true, "Warehouse.enableSchemaTTL") {
-		ttlInMinutes := conf.GetDurationVar(720, time.Minute, "Warehouse.schemaTTLInMinutes")
+		ttlInMinutes := conf.GetDurationVar(720, time.Minute, "Warehouse.schemaTtlInMinutes")
```

Comment:
Previous config (`schemaTTLInMinutes`) seems more appropriate. For abbreviations, it's better to keep it as uppercase.

Reply:
With `schemaTTLInMinutes`, the env key will be `SCHEMA_TTLIN_MINUTES` which I found less readable. So I changed it to `schemaTtlInMinutes` which has env key as `SCHEMA_TTL_IN_MINUTES`

Reply:
I think that should be fine, we don't use env keys for configuring these configs. Now there is inconsistency because at one place we have `enableSchemaTTL` and at the other place we have `schemaTtlInMinutes`.

We can use something like this for test if we want:
```GO
t.Setenv(config.ConfigKeyToEnv(config.DefaultEnvPrefix, "Warehouse.schemaTtlInMinutes"), strconv.Itoa(tc.schemaTTL))
```

Reply:
Reverted the env casing change